docs/Products/omen/index.md

```diff
@@ -6,6 +6,12 @@ has_children: true
 nav_order: 1
 ---
 
-## Welcome to Omen!
+# Welcome to Omen!
 
-[Omen](https://omen.eth.link/) is a fully decentralized prediction market platform built on the [Gnosis conditional token framework](https://docs.gnosis.io/conditionaltokens/). Users can create and/or participate in information markets on various topics (crypto, politics, sports, etc.) and stake their crypto on possible outcomes. Once an Omen market has reached the closing date, anyone can start the process of crowdsourcing the search for the correct answer. This is powered by [reality.eth.link](https://reality.eth.link/). In the case of a dispute, an arbitrator can be called to make a judgment.
\ No newline at end of file
+___
+
+[**Omen**](https://omen.eth.link/) is a fully decentralized multi-chain prediction market platform built on the [Gnosis conditional token framework](https://docs.gnosis.io/conditionaltokens/). Users can create and/or participate in information markets on various topics (crypto, politics, sports, etc.) and stake their crypto on possible outcomes. 
```

Comment:
@KeenanLukeOM Hey Keenan would be good to add in place where there are links. Anchor tag that will lead user to new tab when they click on link since it can be quite annoying to just redirect you. 
Heres example for omen how it should be done:
<strong><a href="https://omen.eth.link/" target="_blank">Omen</a></strong>